.github/workflows/update_licenses.yaml

```
@@ -8,7 +8,7 @@ jobs:
   licensed:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
       with:
         ref: ${{github.ref}} # checkout branch not SHA
```

licensed-ci using `workflow: branch` doesn't work well with actions/checkout@v2 or later without also setting depth to 0.  See the note in that repo's [usage](https://github.com/jonabc/licensed-ci#usage).

```suggestion
        ref: ${{github.ref}} # checkout branch not SHA
        depth: 0
```